_pages/policies/responding-to-public-disclosure-vulnerabilities.md

```diff
@@ -11,9 +11,9 @@ When someone in the public alerts us to a potential vulnerability, we need to re
 
 ## Triage the vulnerability
 
-In PagerDuty, someone is on call to be the **first responder**. This role rotates weekly.
+In order to respond quickly to reports submitted by the public, the Infrastructure team has designated a small group of folks who will monitor incoming reports and initiate response accordingly. This role has been designated **first responder** and will be assigned in [#vuln-disclosure](https://gsa-tts.slack.com/messages/vuln-disclosure/) with an @ (mention). Barring schedule conflicts, the role will rotate weekly.
```

Comment:
Suggest adding "Each response team member should have their email address on the cloud.gov whitelist for outside-of-business-hours alerts; new response team members should ask the cloud.gov Product Lead, Director, or Deputy Director to get added to this list."